components/single_image_view/single_image_view.jsx

```diff
@@ -124,14 +124,14 @@ export default class SingleImageView extends React.PureComponent {
             />
         );
 
-        let imageNameClass = 'image-name';
+        let imageNameClass = 'file-preview__link';
         if (compactDisplay) {
             imageNameClass += ' compact-display';
         }
 
         const fileHeader = (
             <div
-                data-testid='image-name'
+                data-testid='file-preview__link'
```

Comment:
Is this testid being used? After a quick search in the code I found no other references apart of the snapshots.

If it is not being used, I would remove it completely.
If it is being used, the related tests should updated (or at least give a heads up to the test team)

Reply:
I don't really know, but I didn't wanted to mess it up, but makes sense, I'll remove it.

Reply:
The references to this where added here 94d285bc648eac5bf71f29ffffb42969e4639d06 and removed here beb80fb7add37d9e7fff355cd5b37bb1f018ed5c so I guess we can remove it.